src/start/belong-server-base.js

```diff
@@ -4,6 +4,8 @@ import '../lib/node-unhandled-rejection';
 // Setup new relic
 import 'newrelic';
 
+import '../modules/on-start/on-start';
```

Comment:
Why not put this inside presence module? `on-start` doesn't sound like a module when it does things different modules should be doing.